internal/controlplane/handlers_oauth.go

```diff
@@ -39,9 +43,25 @@ import (
 	"github.com/stacklok/minder/internal/db"
 	"github.com/stacklok/minder/internal/engine"
 	"github.com/stacklok/minder/internal/logger"
+	"github.com/stacklok/minder/internal/providers"
 	pb "github.com/stacklok/minder/pkg/api/protobuf/go/minder/v1"
 )
 
+// Track how often users who register a token are correlated with the
+// GitHub user from GetAuthorizationURL
+var tokenMeter = otel.Meter("enrolled-token")
+var tokenMatches metric.Int64Counter
```

Comment:
we have a metrics construct for the controlplane https://github.com/stacklok/minder/blob/main/internal/controlplane/metrics/metrics.go should this be moved there?

Reply:
I put this here in hopes of avoiding as much entanglement across objects, but I've moved it to metrics.  When in Rome...